.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
```

Comment:
Oh, I thought you were already familiar? Found it in `mt-web-client` but not others.
http://editorconfig.org/